### PR TITLE
[15653] Modify Ownership QoS Tutorial

### PR DESCRIPTION
**Pull request by MRicoIE2C**
Tutorial on how to modify Ownership and Ownership Strength QoS within the `ros-demo-nodes-cpp` demo.

### CODE REVIEW

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        <data_reader profile_name="/chatter">
+            <qos>
+                <ownership>
+                    <kind>EXCLUSIVE</kind>
+                </ownership>
+            </qos>
+        </data_reader>
+    </profiles>
+
+For the next section of this tutorial, lets consider both created XML files are stored in `~/` path.
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:78]:
```suggestion
For the next section of this tutorial, let us consider both created XML files are stored in `~/` path.
```

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+Execute ROS 2 demo nodes with modified QoS
+------------------------------------------
+
+Open one terminal and source Vulcanexus environment.
+To set `profiles1.xml` to define the profile configuration used on the creation of ROS 2 nodes,
+it is needed to populate the `FASTRTPS_DEFAULT_PROFILES_FILE` environment variable to point out to the file.
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:85]:
```suggestion
populating the `FASTRTPS_DEFAULT_PROFILES_FILE` environment variable to point out to the file is needed.
```

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+Thus, in the terminal, run the following command:
+
+.. code-block:: bash
+
+    export FASTRTPS_DEFAULT_PROFILES_FILE=~/profiles1.xml
+
+Then, you can run `ros-demo-nodes-cpp` program to create a listener with `EXCLUSIVE_OWNERSHIP_POLICY` QoS:
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:92]: I'd move this sentence to the previous paragraph and combine both the previous snippet and this one into one

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+.. code-block:: bash
+
+    ros2 run demo_nodes_cpp listener
+
+Open another terminal and source Vulcanexus environment.
+To create `ros-demo-nodes-cpp` talker, run the following commands:
+
+.. code-block:: bash
+
+    export FASTRTPS_DEFAULT_PROFILES_FILE=~/profiles1.xml
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:103]: Maybe add the commands to source the installation. Most users will just copy paste without really reading so they'll need that

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    ros2 run demo_nodes_cpp talker
+
+Now both terminals should be communicating.
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:106]: I'd say the nodes are communicating, not the terminals

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+Can be seen that the `Hellow World` messages that talker sends are being received by listener.
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:107]: This sentence lacks a subject

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+The number of those messages coincides.
+
+In a third terminal, source Vulcanexus environment.
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:110]: Again, just add the sourcing into the snippet

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+this time the `FASTRTPS_DEFAULT_PROFILES_FILE` will point out to `profiles2.xml`:
+
+.. code-block:: bash
+
+    export FASTRTPS_DEFAULT_PROFILES_FILE=~/profiles2.xml
+    ros2 run demo_nodes_cpp talker
+
+Now it must be seen that the first talker keeps sending messages,
+but the messages being read by listener are those of the newly created talker
+(The number of the message being sent by last talker should be the same as the number of the arriving message in listener).
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:122]: We should emphasize here that this is because it has a higher strength

**Review by EduPonz — CHANGES_REQUESTED** [review]

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 9]
```diff
@@ -75,50 +72,44 @@ For the `profiles2.xml` file, write down the same code, but changing the value 1
         </data_reader>
     </profiles>
 
-For the next section of this tutorial, lets consider both created XML files are stored in `~/` path.
+For the next section of this tutorial, let us consider both created XML files are stored in `~/` path.
```
Comment (EduPonz) [inline docs/rst/tutorials/core/qos/ownership/ownership.rst:75]:
```suggestion
For the next section of this tutorial, let us consider both created XML files are stored in the ``~/`` directory.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

File: docs/rst/tutorials/core/qos/ownership/ownership.rst [hunk 10]
```diff
@@ -75,50 +72,44 @@ For the `profiles2.xml` file, write down the same code, but changing the value 1
...
 Open one terminal and source Vulcanexus environment.
-To set `profiles1.xml` to define the profile configuration used on the creation of ROS 2 nodes,
-it is needed to populate the `FASTRTPS_DEFAULT_PROFILES_FILE` environment variable to point out to the file.
-Thus, in the terminal, run the following command:
-
-.. code-block:: bash
-
-    export FASTRTPS_DEFAULT_PROFILES_FILE=~/profiles1.xml
-
+To set `small_strength.xml` to define the profile configuration used on the creation of ROS 2 nodes, populating the `FASTRTPS_DEFAULT_PROFILES_FILE` environment variable to point out to the file is needed.
```
[the remark this answers is not in the document]
Comment (MRicoIE2C) [reply docs/rst/tutorials/core/qos/ownership/ownership.rst:81]: I have added a note below the creation of both.

**Review by EduPonz — APPROVED** [review]

**Pull request merged**